docker/compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+  web:
+    build:
+      context: ../
+      dockerfile: docker/Dockerfile
+    command:
+      - /bin/bash
+      - -c
+      - |
+        npm install
+        npm start
+    environment:
+      - RESOURCE_CONFIG=${RESOURCE_CONFIG}
+      - OPERATION_CONFIG=${OPERATION_CONFIG}
```

Comment:
Is it too complicated right here?
If docker run then people can use `-e`, and if docker-compose people can just define env var?

Reply:
With docker run, we need to run multiple command to build the image and start, but with compose file its easy to do it. Example 
```
PORT=8080 RESOURCE_CONFIG=configs/resources/sample-resource.yml OPERATION_CONFIG=configs/operations/github-merged-pulls-monitor.yml docker-compose -p web-1 up -d

PORT=8081 RESOURCE_CONFIG=configs/resources/sample-resource.yml OPERATION_CONFIG=configs/operations/github-workflow-runs-monitor.yml docker-compose -p web-2 up -d
```

Reply:
Eventually it will just be a docker run once we have the official automation app image build and released.